### PR TITLE
[KIECLOUD-716] -  merge main into release-v7.13.x

### PR DESCRIPTION
**Pull request by spolti**

### CODE REVIEW

File: deploy/olm-catalog/dev/7.13.2-1/manifests/businessautomation-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -409,7 +409,11 @@ spec:
   - operator
   labels:
     alm-owner-businessautomation: businessautomation-operator
+<<<<<<< HEAD
```
Comment (bacciotti) [inline deploy/olm-catalog/dev/7.13.2-1/manifests/businessautomation-operator.clusterserviceversion.yaml:412]: It looks this can't be here, right?

**Review by bacciotti — CHANGES_REQUESTED** [review]
I would recommend removing some git terms in the code, please.

**Review by desmax74 — APPROVED** [review]

**Review by bacciotti — APPROVED** [review]
LGTM

**Review by lampajr — APPROVED** [review]

**Review by desmax74 — APPROVED** [review]

**Review by davidesalerno — APPROVED** [review]

**Pull request merged**